### ISSUE TITLE
Not able to get the element

### ISSUE THREAD

**Issue by ritusenleo** [microsoft/WinAppDriver#1925]
Trying to get the value of  titlebar of the page :
 var pageTitle = _driver.Current.FindElementByAccessibilityId("TitleBar").GetAttribute("Value.Value");

Also tried :  var pageTitle = _driver.Current.FindElementByTagName("TitleBar").GetAttribute("Value.Value");

In both the case it is throwing the following error:
error: The HTTP request to the remote WebDriver server for URL http://127.0.0.1:4723/session/ABC7DE2E-34C5-444B-A202-DD1B54E6774A/element timed out after 60 seconds. (62.8s)

Suprising thing is  : This code used to work before , suddenly stopped working where I have done no changes in the code.

Team please help me.

**Comment by ritusenleo** [comment]
@anunay1 , I do  confirm from the respective Dev team nothing is changed in that particular section ... and i have not change anything in the framework....
If any more suggestion to me... it will be helpful

**Comment by anunay1** [comment]
What do you get in page source?

**Comment by ritusenleo** [comment]
Code is failing when trying to get the page source:
var PageSource= _driver.Current.PageSource;
error: The HTTP request to the remote WebDriver server for URL http://127.0.0.1:4723/session/7DBCF40E-97CC-41E4-A997-D0F2FFA1E932/source timed out after 60 seconds.

**Comment by anunay1** [comment]
then there is some issue with the winappdriver server, what is shown in the console output of winappdriver. Also check before this line are you able to ping the server.

**Comment by ritusenleo** [comment]
@anunay1 I am able to do everything except this line... 
In the code I just need to get the title of the page, which really used to work but suddenly providing the following sort of error.

 error: The HTTP request to the remote WebDriver server for URL http://127.0.0.1:4723/session/7DBCF40E-97CC-41E4-A997-D0F2FFA1E932/source timed out after 60 seconds.

**Comment by anunay1** [comment]
the error says that it timed out, that means the client request is not able to reach the server.

**Comment by ritusenleo** [comment]
Yes, but it is so wierd ... If I am trying to click any other element it is working, whenever I am trying to get the titlebar it is providing the server error.. any suggestion from your end

**Comment by ritusenleo** [comment]
It is solved:
_driver.Current.Title
